src/selectors/contents.js

```diff
@@ -64,21 +82,28 @@ export const getDirectories = createSelector([getContentsPerDirectories], conten
   ),
 )
 
-export const getFavoritesContents = createSelector([getAllContents], contents =>
-  contents.filter(c => c.isFavorite),
+export const getAugmentedFavoriteContents = createSelector(
+  [getAllContents, getFavorites],
+  (contents, favorites) =>
+    getAugmentedContents(contents.filter(content => favorites[content.id]), favorites),
 )
 
-export const getHistoryContents = createSelector([getAllContents], contents =>
-  contents
-    .filter(content => content.lastPlayed)
-    .sort((c1, c2) => (c1.lastPlayed > c2.lastPlayed ? -1 : 1)),
+export const getAugmentedHistoryContents = createSelector(
+  [getAllContents, getFavorites, getHistory],
+  (contents, favorites, history) =>
+    getAugmentedContents(
+      contents
+        .filter(content => history[content.id])
+        .sort((c1, c2) => (c1.lastPlayed > c2.lastPlayed ? -1 : 1)),
+      favorites,
+    ),
 )
 
-export const getPlayingContent = state => state.playlist.playingContent
+const getPlayingContent = state => state.playlist.playingContent
 
-export const getMyPlaylistContents = createSelector(
-  [getPlaylistContents, getUsername],
-  (contents, username) => contents.filter(c => c.username === username),
+export const getAugmentedPlayingContent = createSelector(
+  [getPlayingContent, getFavorites],
+  (content, favorites) => content && getAugmentedContent(content, favorites),
```

Comment:
If the goal of `content &&` here is to not call the function on `undefined` or `null`, I think this comes in conflict with 
```javascript
  static propTypes = {
    playingContent: PropTypes.object,

Reply:
`PropTypes.object` isn't `PropTypes.object.isRequired`. In fact, there's a check in MainScreen.js which would break if we had a `playingContent` as an empty object : 

```js
{playingContent && (<PlaylistStatusBar />)}
```

it would display the status bar even with no playing content